py/desitarget/fortran/README

```diff
@@ -0,0 +1,17 @@
+The files in this directory are:
+
+ADM-v1dump.f: FORTRAN code to parse UCAC/URAT files, adapted from
+              http://cdsarc.u-strasbg.fr/ftp/I/329/URAT1/access/v1dump.f
+              by Adam D. Myers (University of Wyoming).
+
+v1sub.f: Routines to help parse UCAC/URAT files, taken directly from
+         http://cdsarc.u-strasbg.fr/ftp/I/329/URAT1/access/v1sub.f
+
+v1dump: An executable compiled at NERSC via:
```

Comment:
You probably didn't intend to add the compiled v1dump file to git.  Please remove it.
(Adding this comment to the README, where it is good to mention the executable, since I can't seem to github comment directly on a binary file itself).

Reply:
Ooops, sorry. Should I blast that with the BFG?

Reply:
It's only 30k so not super critical to blast with the BFG, but if you are comfortable with the BFG and want to use it, go for it.

Reply:
In the end I decided to be lazy as it's such a small file and will never make it into master. I'll blast it down the road if I feel energetic.

Reply:
Oh, I did remove it though.